spec/services/service_offering/add_to_portfolio_item_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe ServiceOffering::AddToPortfolioItem do
```

Comment:
@syncrou @lindgrenj6 
Shouldn't this go into the service_catalog folder?

Reply:
@lindgrenj6 I agree with @mkanoor that this should be moved to match the folder layout in the main repo.

Reply:
In reference to this comment: https://github.com/ManageIQ/service_portal-api/pull/97#discussion_r252491841 ( the resolution of will determine the answer here. )